lib/app/components/pages/home/controllers/home_controller.dart

```diff
@@ -40,6 +49,44 @@ class HomeController extends GetxController {
     }
   }
 
+  void initTab(List<TabData> list) {
+    tabDataList.clear();
+    tabDataList.addAll(list);
```

Comment:
```suggestion
    tabDataList.assignAll(list);
```
`assignAll`でclearとaddAllまとめて同じ処理なのでこちらに変えましょうか。

Reply:
対応いたしました。